assignments/arrays.js

```diff
@@ -63,32 +63,59 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 2011 Jeep Wrangler`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// let lastCar = 0;
+// console.log(inventory[inventory.length-1]);
+console.log("Make is Lincoln, and model is Town Car");
```

Comment:
A flexible way to write this would be the following:

console.log("make is " + inventory[inventory.length -1].car_make + " and the model is " + inventory[inventory.length - 1].car_model)

It's good to get in the habit of writing out messages this way. It will make your code more flexible.